pkg/processor/metricfrequencyprocessor/config_test.go

```diff
@@ -0,0 +1,27 @@
+package metricfrequencyprocessor
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/configtest"
+	"path"
+	"testing"
```

Comment:
```suggestion
	"path"
	"testing"
	
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
	"go.opentelemetry.io/collector/component/componenttest"
	"go.opentelemetry.io/collector/config"
	"go.opentelemetry.io/collector/config/configtest"
```

adding a linter for that is long overdue